File: source codes/Lab 2 Sasha Angermueller/Rectangle.cs
// Class:	CSE 1321L
// Section: 	08         
// Term:		Fall 2018  
// Instructor:	
// Name:		Sasha Angermueller
// Lab#:	    2

using System;

namespace Rectangle
{
    class Rectangle
    {
        static void Main(string[] args)
        {
            int width,height,area, perimiter;
            width = 4;
            height = 8;
            area = width * height;
            perimiter = (width * 2) + (height * 2);
            Console.WriteLine("The width: " + width);
            Console.WriteLine("The height: " + height);
            Console.WriteLine("The area: " + area);
            Console.WriteLine("The perimeter: " + perimiter);
            Console.ReadLine();
        }
    }
}

File: School source codes/Lab 10/Rectangle.rb
class Rectangle
  def initialize(height,width)
    @height,@width = height,width
  end
  def height
    @height
  end
  def width
    @width
  end
  def setheight(double)
    @height = double
  end
  def setwidth(double)
    @width = double
  end
  def area
    @height*@width
  end
  def perimeter
    (@height*2)+(@width*2)
  end
end
a = Rectangle.new(1,5)
b = Rectangle.new(5,1)
#a.setheight(5)
#a.setwidth(6)
#b.setheight(2)
#b.setwidth(8)
puts "First rectangle area #{a.area}"
puts "First rectangle perimeter #{a.perimeter}"
puts "Second rectangle area #{b.area}"
puts "Second rectangle perimeter #{b.perimeter}"
